### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DimkaVeselov**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,54 +7,54 @@
     <title>Tesla</title>
 </head>
 <body>
-<div class="wrap">
-    <!-- шапка -->
```
Comment (KsuBurn) [inline index.html]: Комментариев быть не должно) Тем более таких очевидных)

File: index.html [hunk 2]
```diff
@@ -7,54 +7,54 @@
...
-                    <a href="#" class="model">Energy</a>
-                </div>
-                <div class="header-menu">
-                    <a href="#" class="header-menu__item">Магазин</a>
-                    <a href="#" class="header-menu__item">Войти</a>
-                </div>
-                <div class="burger">
-                    <a href="#" class="burer-btn"><img src="img/burger.png" alt="Burger menu"></a>
-                </div>
-    </div>
```
Comment (KsuBurn) [inline index.html]: Почему не используешь теги по смыслу? Для навигации nav, для шапки header

File: index.html [hunk 3]
```diff
@@ -7,54 +7,54 @@
...
-        <!-- контент -->
-    <div class="content">
-        <!-- первая секция -->
-        <div class="first-floor-one">
```
Comment (KsuBurn) [inline index.html]: название класса должно отображать, что находится в данном блоке)

File: index.html [hunk 4]
```diff
@@ -7,54 +7,54 @@
...
-                    </div>
-                    <div class="stat__item-two">
-                        <img src="img/Test_drive2.png" alt="test drive"> 
-                        <button class="btn__first-floor">Заказать</button>
-                    </div>
-                </div>
-                <div class="swipe-down_btn"><input type="image" src="img/swipe-down.svg"></div>
-            </div>
-        </div>
+    <div class="wrap">
```
Comment (KsuBurn) [inline index.html]: Здесь тоже непонятно по классу, что лежит в этом блоке)

File: index.html [hunk 5]
```diff
@@ -82,81 +82,80 @@ <h3>Будущее вождения</h3>
...
-                        </div>
+        <div class="safety-info">
+            <div class="description-safety">
+                <h3>Самый безопасный автомобиль в мире</h3>
+                <p>Модель S достигла наивысшей оценки безопасности любого автомобиля, когда-либо испытанного, и установила рекорд по
+                    наименьшей вероятности травмы пассажиров - от несчастных случаев на переднем, боковом, заднем сидении и в ДТП с
+                    переворотом авто.</p>
+            </div>
+            <div class="rating-safety">
+                <h3>Оценка защиты:</h3>
```
Comment (KsuBurn) [inline index.html]: нет класса

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 37
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648&bid=17996861)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 37
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648&bid=17996861)

File: index.html (hunk 5 above)
Comment (DimkaVeselov) [reply index.html]: Принял, пробегусь исправлю

File: styles/fonts.css [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  src: 
+    url('/fonts/Montserrat-SemiBold.ttf') format('truetype'),
+    url('/fonts/montserrat-semibold-webfont.woff') format('woff'),
+    url('/fonts/montserrat-semibold-webfont.woff2') format('woff2');
+	font-weight: 600;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
```
Comment (KsuBurn) [inline styles/fonts.css:32]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 37
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648&bid=17996861)

File: styles/style.css [hunk 7]
```diff
@@ -0,0 +1,699 @@
...
+.btn__first-floor:hover {
+  color: #fff;
+  background: #E51937;
+  border-color: #E51937;
+}
+
+.swipe-down_btn {
+  position: relative;
+  left: 8px;
+  transition-duration: .9s;
```
Comment (KsuBurn) [inline styles/style.css:214]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    url('/fonts/Montserrat-Medium.ttf') format('truetype'),
+    url('/fonts/montserrat-medium-webfont.woff') format('woff'), 
+    url('/fonts/montserrat-medium-webfont.woff2') format('woff2');
+	font-weight: 500;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src:
```
Comment (KsuBurn) [inline styles/fonts.css:23]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 9]
```diff
@@ -0,0 +1,699 @@
...
+.whs-img{
+  width: 94px;
+  position: relative;
+  top: 17vw;
+  left: 58vw;
+}
+
+@media screen and (max-width: 1080px) {
+
+  .header-models {
```
Comment (KsuBurn) [inline styles/style.css:440]: Codacy Issue found: [Unexpected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 10]
```diff
@@ -0,0 +1,699 @@
...
+ 
+.btn__first-floor {
+  text-transform: uppercase;
+  display: block;
+  border: 2px solid #E51937;
+  color: #E51937;
+  font-weight: bold;
+  font-size: 13px;
+  font-style: normal;
+  letter-spacing: .5px;
```
Comment (KsuBurn) [inline styles/style.css:198]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 11]
```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'montserrat';
+  }
```
Comment (KsuBurn) [inline styles/style.css:6]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 8 above)
Comment (KsuBurn) [inline styles/fonts.css:23]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 12]
```diff
@@ -0,0 +1,699 @@
...
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  margin-left: 10px;
+  padding: 10px 43px; 
+  border: 2px solid #fff;
+  background: #E51937;
```
Comment (KsuBurn) [inline styles/style.css:406]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src: 
+    url('/fonts/Montserrat-SemiBold.ttf') format('truetype'),
+    url('/fonts/montserrat-semibold-webfont.woff') format('woff'),
+    url('/fonts/montserrat-semibold-webfont.woff2') format('woff2');
+	font-weight: 600;
```
Comment (KsuBurn) [inline styles/fonts.css:27]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 14]
```diff
@@ -0,0 +1,699 @@
...
+  }
+
+  .btn__first-floor {
+    background-color: #E51937;
+    color: #fff;
+  }
+}
+
+@media screen and (max-width: 770px) {
+   .nav {
```
Comment (KsuBurn) [inline styles/style.css:544]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 15]
```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'montserrat';
+  }
+
+a {
+	color: #000000;
```
Comment (KsuBurn) [inline styles/style.css:9]: Codacy Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 16]
```diff
@@ -0,0 +1,699 @@
...
+
+.btn-safety {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  margin-left: 10px;
+  padding: 10px 43px;
```
Comment (KsuBurn) [inline styles/style.css:404]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src:
+    url('/fonts/Montserrat-Bold.ttf') format('truetype'),
+    url('/fonts/montserrat-bold-webfont.woff') format('woff'),
+    url('/fonts/montserrat-bold-webfont.woff2') format('woff2');
+	font-weight: bold;
```
Comment (KsuBurn) [inline styles/fonts.css:37]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 18]
```diff
@@ -0,0 +1,699 @@
...
+
+.safety-img {
+  background-image: url(../img/Model_S-1.png);
+  background-repeat: no-repeat;
+  background-position-x: 104%;
+  background-position-y: 25%;
+  background-size: 85%;
+}
+
+.whs-img{
```
Comment (KsuBurn) [inline styles/style.css:431]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 19]
```diff
@@ -0,0 +1,39 @@
...
+	font-style: normal;
```
Comment (KsuBurn) [inline styles/fonts.css:38]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 20]
```diff
@@ -0,0 +1,699 @@
...
+  padding: 10px 43px; 
+  border: 2px solid #fff;
+  background: #E51937;
+  cursor: pointer;
+  transition-duration: .4s;
+}
+
+.btn-safety:hover {
+  background: #fff;
+  color: #E51937;
```
Comment (KsuBurn) [inline styles/style.css:413]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 21]
```diff
@@ -0,0 +1,699 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+.star-item span {
+  font-size: 10px;
+  line-height: 12px;
+  opacity: .6;
```
Comment (KsuBurn) [inline styles/style.css:360]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 22]
```diff
@@ -0,0 +1,699 @@
...
+  }
+
+  .stat__btn {
+    right: 80px;
+  }
+}
+
+@media screen and (max-width: 375px) {
+
+  .nav {
```
Comment (KsuBurn) [inline styles/style.css:623]: Codacy Issue found: [Unexpected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 23]
```diff
@@ -0,0 +1,699 @@
...
+  .test-drive {
+    display: none;
+  }
+
+  .test-drive__red {
+    display: block;
+  }
+
+  .btn__first-floor {
+    background-color: #E51937;
```
Comment (KsuBurn) [inline styles/style.css:538]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 24]
```diff
@@ -0,0 +1,699 @@
...
+
+.title-text {
+  font-size: 89px;
+  font-weight: 600;
+  margin: 0 auto;
+  padding-bottom: 16px;
+  position: relative;
+  top: 11px;
+  left: 9px;
+  letter-spacing: .6px;
```
Comment (KsuBurn) [inline styles/style.css:112]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 25]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src: 
+    url('/fonts/Montserrat-Medium.ttf') format('truetype'),
+    url('/fonts/montserrat-medium-webfont.woff') format('woff'), 
+    url('/fonts/montserrat-medium-webfont.woff2') format('woff2');
+	font-weight: 500;
```
Comment (KsuBurn) [inline styles/fonts.css:17]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 26]
```diff
@@ -0,0 +1,39 @@
+@font-face {
+	font-family: 'montserrat';
+  src:
+    url('/fonts/Montserrat-Regular.ttf') format('truetype'),
+    url('/fonts/montserrat-regular-webfont.woff') format('woff'),
+    url('/fonts/montserrat-regular-webfont.woff2') format('woff2');
+	font-weight: normal;
```
Comment (KsuBurn) [inline styles/fonts.css:7]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 27]
```diff
@@ -0,0 +1,699 @@
...
+.test-drive__red {
+  display: none;
+}
+
+.stat__btn {
+  position: relative;
+  top: 11px;
+  right: 53px;
+}
+
```
Comment (KsuBurn) [inline styles/style.css:189]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 28]
```diff
@@ -0,0 +1,699 @@
...
+  padding: 0;
+  font-family: 'montserrat';
+  }
+
+a {
+	color: #000000;
+  text-decoration: none;
+}
+
+.wrapper  {
```
Comment (KsuBurn) [inline styles/style.css:13]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css [hunk 29]
```diff
@@ -0,0 +1,39 @@
...
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src: 
+    url('/fonts/Montserrat-Medium.ttf') format('truetype'),
+    url('/fonts/montserrat-medium-webfont.woff') format('woff'),
```
Comment (KsuBurn) [inline styles/fonts.css:15]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 33
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648&bid=17996861)

File: styles/fonts.css (hunk 6 above)
Comment (KsuBurn) [inline styles/fonts.css:32]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 30]
```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'montserrat';
```
Comment (KsuBurn) [inline styles/style.css:5]: Codacy Issue found: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 8 above)
Comment (KsuBurn) [inline styles/fonts.css:23]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 11 above)
Comment (KsuBurn) [inline styles/style.css:6]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 8 above)
Comment (KsuBurn) [inline styles/fonts.css:23]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 29 above)
Comment (KsuBurn) [inline styles/fonts.css:15]: Codacy Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 12 above)
Comment (KsuBurn) [inline styles/style.css:406]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 13 above)
Comment (KsuBurn) [inline styles/fonts.css:27]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 14 above)
Comment (KsuBurn) [inline styles/style.css:544]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 15 above)
Comment (KsuBurn) [inline styles/style.css:9]: Codacy Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 16 above)
Comment (KsuBurn) [inline styles/style.css:404]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 17 above)
Comment (KsuBurn) [inline styles/fonts.css:37]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 18 above)
Comment (KsuBurn) [inline styles/style.css:431]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 19 above)
Comment (KsuBurn) [inline styles/fonts.css:38]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 20 above)
Comment (KsuBurn) [inline styles/style.css:413]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 31]
```diff
@@ -0,0 +1,699 @@
...
+  .params {
+    display: flex;
+    flex-flow: column;
+  }
+
+  .stat-power {
+    margin-right: 160px;
+  }
+
+  .params {
```
Comment (KsuBurn) [inline styles/style.css:517]: Codacy Issue found: [Unexpected duplicate selector ".params", first used at line 508 (no-duplicate-selectors)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 16 above)
Comment (KsuBurn) [inline styles/style.css:404]: Codacy Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 25 above)
Comment (KsuBurn) [inline styles/fonts.css:17]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 32]
```diff
@@ -0,0 +1,699 @@
...
+  background: #fff;
+  padding: 10px 43px;
+  transition-duration: 0.4s;
+  cursor: pointer;
+}
+
+.btn__first-floor:hover {
+  color: #fff;
+  background: #E51937;
+  border-color: #E51937;
```
Comment (KsuBurn) [inline styles/style.css:208]: Codacy Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 26 above)
Comment (KsuBurn) [inline styles/fonts.css:7]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 27 above)
Comment (KsuBurn) [inline styles/style.css:189]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 28 above)
Comment (KsuBurn) [inline styles/style.css:13]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 15 above)
Comment (KsuBurn) [inline styles/style.css:9]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/fonts.css (hunk 29 above)
Comment (KsuBurn) [inline styles/fonts.css:15]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 33]
```diff
@@ -0,0 +1,699 @@
...
+  display: block;
+  border: 2px solid #E51937;
+  color: #E51937;
+  font-weight: bold;
+  font-size: 13px;
+  font-style: normal;
+  letter-spacing: .5px;
+  background: #fff;
+  padding: 10px 43px;
+  transition-duration: 0.4s;
```
Comment (KsuBurn) [inline styles/style.css:201]: Codacy Issue found: [Unexpected leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648&bid=17996861)

File: styles/style.css [hunk 34]
```diff
@@ -0,0 +1,687 @@
...
+}
+
+.btn-info {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  padding: 10px 43px;
```
Comment (KsuBurn) [inline styles/style.css:295]: Codacy Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 35]
```diff
@@ -0,0 +1,687 @@
...
+  margin-right: 5px;
+}
+
+.model:hover {
+  color: #e51937;
+  border-bottom: 2px solid;
+  padding: 0 0 6px 0;
+}
+
+.model-s__link {
```
Comment (KsuBurn) [inline styles/style.css:62]: Codacy Issue found: [Expected selector ".model-s__link" to come before selector ".model-s__link:first-child" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css (hunk 34 above)
Comment (KsuBurn) [inline styles/style.css:295]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

File: styles/style.css [hunk 36]
```diff
@@ -0,0 +1,687 @@
...
+
+  .stat__btn {
+    top: 50px;
+    left: 10px;
+  }
+
+  .swipe-down {
+    padding-top: 25px;
+  }
+}
```
Comment (KsuBurn) [inline styles/style.css:687]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)

**Pull request merged**